workspaces/src/operations.rs

```diff
@@ -236,19 +236,19 @@ impl<'a> Transaction<'a> {
 
 /// Similiar to a [`Transaction`], but more specific to making a call into a contract.
 /// Note, only one call can be made per `CallTransaction`.
-pub struct CallTransaction<'a, 'b> {
+pub struct CallTransaction<'a> {
     worker: &'a Worker<dyn Network>,
     signer: InMemorySigner,
     contract_id: AccountId,
-    function: Function<'b>,
+    function: Function,
 }
 
-impl<'a, 'b> CallTransaction<'a, 'b> {
+impl<'a, 'b> CallTransaction<'a> {
```

Comment:
isn't `'b` unused?

Reply:
I'm surprised there weren't any warnings for this both from rustc and clippy